### PR TITLE
Created POST endpoint for message popup questions

### PR DESCRIPTION
**Pull request by david-rosales**
I set up the post endpoint for the database table holding the questions.

It's a very small change but I just want to make sure it's not going to kill the database should I use it.

### CODE REVIEW

**Comment by david-rosales** [comment]
@kevinrobinson Can you review this?

File: ui/src/helpers/api.js [hunk 1]
```diff
@@ -76,4 +76,12 @@ export function questionsQuery() {
   return request
     .get('/server/questions')
     .set('Content-Type', 'application/json');
+}
+
+export function saveQuestions(record) {
```
Comment (kevinrobinson) [inline ui/src/helpers/api.js]: Should `record` here be `questions`, based on how the server interprets the request body?

**Comment by kevinrobinson** [comment]
@david-rosales One question on the request body format, but this looks good!

File: ui/src/helpers/api.js (hunk 1 above)
Comment (david-rosales) [inline ui/src/helpers/api.js]: Good point. I updated it but am wondering whether this has any actual effect on the underlying database code or is it just for readability?

Comment (kevinrobinson) [inline ui/src/helpers/api.js]: Good question!  Just clarify, we could call it `variableNamesOnlyMatterToHumans`:)  But clarity in naming and documentation is usually particularly important when crossing system boundaries like here (eg., from the UI in the browser to the server code).

Comment (david-rosales) [inline ui/src/helpers/api.js]: Sounds good, merging now.

**Pull request merged**